packages/eslint-config-node/readme.md

```diff
@@ -28,6 +28,46 @@ Use this ruleset in conjunction with any of the above version-specific rulesets.
 
 This ruleset includes rules which deal with how the code looks like and not how it works. It helps keeping the code clean and consistent. 🎨
 
+## STRV Config
+
+package.json
```

Comment:
I am personally a huge fan of explicit config files per tool. _package.json_ should be only used for declaring the package's requirements, not as a place to store configuration options.